pom.xml

```diff
@@ -20,14 +20,15 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <pubsub.client.version>2.4.0</pubsub.client.version>
     <commons.lang.version>2.6</commons.lang.version>
     <logger.version>1.7.25</logger.version>
     <jackson.version>2.10.5.1</jackson.version>
     <junit.version>4.13.1</junit.version>
-    <vertx-unit.version>4.0.0</vertx-unit.version>
+    <vertx.version>4.0.0</vertx.version>
     <wiremock.version>2.27.2</wiremock.version>
+    <raml-module-builder.version>33.2.2</raml-module-builder.version>
```

Comment:
Could you, please, update rmb version to 33.2.3?

Reply:
done